### DISCUSSION TITLE
Responsive virtualized table

### DISCUSSION THREAD

**Discussion by vsSanti** [TanStack/table#1999] [General]
I'm trying to build a responsive table using virtualized rows (since I have thousands of them) that occupy the whole available width of the page, but I'm not being able to achieve it.
I have already used react-virtualized-auto-sizer packageand it works great for the content, but not the headers, wich seems to have a standard width of 150px. Has anyone done it yet?
BTW, I'm using the v7 version.

**Comment by matthewma7** [comment]
I created a simpler useFlexLayout here and it is working for me. 
Disclaimer: I am new to the library as well.
https://github.com/tannerlinsley/react-table/discussions/2022
